gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
-agp = "7.3.1"
+agp = "8.0.0"
```

Comment:
AGP 8.0.0 and its Kotlin Plugin started to support Kotlin version 1.8.x

Reply:
Could you use 8.0.2 instead?

Reply:
@yshrsmz sure, let me try this path release 8.0.2

Reply:
@yshrsmz it's okay now.  We are using 8.0.2